### ISSUE TITLE
Invalid webtask created when package.json contains no dependencies

### ISSUE THREAD

**Issue by toboid** [auth0/wt-cli#133]
If you create webtask that has a package.json with no dependencies, the created webtask is invalid. 

I have created an example [here](https://github.com/toboid/wt-cli/tree/webtask-no-deps/sample-webtasks/webtask-no-deps)

When deployed with `wt create .` the resulting webtask raises the following error when executed:
```
{
  "code": 400,
  "error": "GET request must specify code to execute using URL."
}
```

One instance where this might occur is when using some devDependencies for testing. 

It's possible to work around using the `--ignore-package-json` flag.

**Comment by toboid** [comment]
If you'll take a PR then i'd be happy to take look at this

**Comment by ggoodman** [comment]
Closed via #134

**Comment by Ahhhz** [comment]
TY I needed this answer.
